### PR TITLE
Changelog for v2.181.0

### PR DESCRIPTION
**Pull request by vEpiphyte**

### CODE REVIEW

File: docs/synapse/userguides/model_updates/update_v2_181_0.rst [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    When used with a ``it:dev:repo`` and an ``inet:url`` node, the edge
+    indicates the repo has content hosted at the URL.
+
+
+``uses``
+    When used with a ``risk:tool:software`` and an ``math:algorithm`` node, the
+    edge indicates the tool uses the algorithm.
+
+
+``uses``
```
Comment (mikemoritz) [inline docs/synapse/userguides/model_updates/update_v2_181_0.rst:82]: This seems like it should be grouped with "uses" above it?

File: docs/synapse/userguides/model_updates/update_v2_181_0.rst [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+Light Edges
+***********
+
+``generates``
+    When used with a ``math:algorithm`` node, the edge indicates the target
+    node was generated by the algorithm.
+
+
+``has``
+    When used with a ``it:dev:repo`` and an ``inet:url`` node, the edge
```
Comment (Cisphyx) [inline docs/synapse/userguides/model_updates/update_v2_181_0.rst:73]:
```suggestion
    When used with an ``it:dev:repo`` and an ``inet:url`` node, the edge
```

File: docs/synapse/userguides/model_updates/update_v2_181_0.rst [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+    indicates the repo has content hosted at the URL.
+
+
+``uses``
+    When used with a ``risk:tool:software`` and an ``math:algorithm`` node, the
```
Comment (Cisphyx) [inline docs/synapse/userguides/model_updates/update_v2_181_0.rst:78]:
```suggestion
    When used with a ``risk:tool:software`` and a ``math:algorithm`` node, the
```

File: docs/synapse/userguides/model_updates/update_v2_181_0.rst [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+    edge indicates the tool uses the algorithm.
+
+
+``uses``
+    When used with a ``file:bytes`` and an ``math:algorithm`` node, the edge
```
Comment (Cisphyx) [inline docs/synapse/userguides/model_updates/update_v2_181_0.rst:81]:
```suggestion
    When used with a ``file:bytes`` and a ``math:algorithm`` node, the edge
```

File: docs/synapse/userguides/model_updates/update_v2_181_0.rst [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+    indicates the file uses the algorithm.
+
+    When used with a ``it:prod:softver`` and an ``math:algorithm`` node, the
```
Comment (Cisphyx) [inline docs/synapse/userguides/model_updates/update_v2_181_0.rst:84]:
```suggestion
    When used with an ``it:prod:softver`` and a ``math:algorithm`` node, the
```

File: docs/synapse/userguides/model_updates/update_v2_181_0.rst (hunk 2 above)
Comment (Cisphyx) [reply docs/synapse/userguides/model_updates/update_v2_181_0.rst]: 'a' before consonant sound, 'an' before vowel sound. Adjusting the generator to use 'an' before a vowel character would probably get it right most of the time.

Comment (vEpiphyte) [reply docs/synapse/userguides/model_updates/update_v2_181_0.rst:73]: I'm tracking that change separately.

**Review by Cisphyx — APPROVED** [review]

**Pull request merged**